### PR TITLE
[Windows] Fix import loops

### PR DESCRIPTION
**Pull request by gpotter2**
Fixes https://github.com/secdev/scapy/issues/434

Just to explain what i did:
- Having the consts.py inside the arch/ folder isn't easy to use: a lot of stuff loaded from the arch/ folder requires to know which operating system is used. So i moved the file out of arch/
- Removed useless import [here](https://github.com/secdev/scapy/pull/435/files#diff-a9f72b412e9acf78502c904846ff2c67L19)
- Redirected several main modules (sendrecv, route) to consts.py (instead of arch/consts.py) so they could be loaded without arch.
- Added a check inside arch/, to avoid loading `compatibility.py` if arch/ was imported from `route.py` (because route.py does not need it, but compatibility.py needs route.py so it created an import loop)

### CODE REVIEW

**Comment by p-l-** [comment]
Thanks @gpotter2. Is it really necessary to keep `arch.consts`? If not, I'd rather have it removed.

**Comment by gpotter2** [comment]
@guedou Codecoverage does not run Windows or other operating systems, we'll have to fix that

**Comment by guedou** [comment]
@gpotter2  I will finish PR#426 by the end of the week. Then, I will make it work on AppVeyor later. So far, its works on Linux and OS X. I still need to fix some broken tests. As of today, codecov supports is WIP.

**Comment by gpotter2** [comment]
@guedou thank you.

So please ignore the failure of the code coverage test...

@p-I- done !

File: scapy/arch/__init__.py [hunk 1]
```diff
@@ -93,7 +66,13 @@ def get_if_hwaddr(iff):
     from scapy.arch.solaris import *
 elif WINDOWS:
     from scapy.arch.windows import *
-    from scapy.arch.windows.compatibility import *
+    # import only if parent is not route.py
```
Comment (guedou) [inline scapy/arch/__init__.py]: Could add a more specific comment to explain why this is required ?

**Comment by guedou** [comment]
I tested `doc/notebooks/graphs-ipids.ipynb` and it worked fine !

Except my small comment, this PR is good to be merged.

File: scapy/arch/__init__.py (hunk 1 above)
Comment (gpotter2) [inline scapy/arch/__init__.py]: Done

**Pull request merged**